### ISSUE TITLE
Adversarially-Trained Deep Nets Transfer Better 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#49]
# TL;DR
Compared to models trained normally, models trained with adversarial noise performed better in transfer learning. The results of the visualization showed that they were classifying in a more human-like sense, which may have influenced the results.
![Source dataset](https://user-images.githubusercontent.com/24524018/90625869-2c352780-e255-11ea-841f-49ed72b5c08f.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.05869

# Submission Dates(yyyy/mm/dd)
2020/07/11

# Authors and institutions
Francisco Utrera, Evan Kravitz, N. Benjamin Erichson, Rajiv Khanna, Michael W. Mahoney
- University of California, Berkeley 

# Methods

# Results
![natural](https://user-images.githubusercontent.com/24524018/90625880-30614500-e255-11ea-9ebf-c7c5faffe60a.png)
![Natural](https://user-images.githubusercontent.com/24524018/90625896-39521680-e255-11ea-9843-4dd631c8abf5.png)

# Comments
